File: Homework/employee/src/employee/Employee.java
package employee;

public class Employee {
	private String id;
	private String name;
	
	Employee(String name, String id){
		this.id = id;
		this.name = name;
	}
	
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name=name;
	}
	
	
	
}	

File: Frontend/Typescript/dislpayclassdata/employee.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Employee = void 0;
var Employee = /** @class */ (function () {
    function Employee(id, name, department, age) {
        this.id = id;
        this.name = name;
        this.department = department;
        this.age = age;
    }
    Employee.prototype.getId = function () {
        return this.id;
    };
    Employee.prototype.getName = function () {
        return this.name;
    };
    Employee.prototype.getDepartment = function () {
        return this.department;
    };
    Employee.prototype.getAge = function () {
        return this.age;
    };
    Employee.prototype.setDepartment = function (department) {
        this.department = department;
    };
    Employee.prototype.setAge = function (age) {
        this.age = age;
    };
    Employee.prototype.setid = function (id) {
        this.id = id;
    };
    Employee.prototype.setName = function (name) {
        this.name = name;
    };
    return Employee;
}());
exports.Employee = Employee;
